src/tabs/gps.html

```diff
@@ -45,104 +45,218 @@
                         </table>
                     </div>
                 </div>
-            </div>
-            <div class="grid-col col3">
+
                 <div class="gui_box grey">
                     <div class="gui_box_titlebar">
                         <div class="spacer_box_title" i18n="gpsSignalStrHead"></div>
                     </div>
                     <div class="spacer_box GPS_signal_strength">
                         <table class="cf_table">
                             <tr class="titles">
-                                <td style="width: 20%;" i18n="gpsSignalSatId"></td>
-                                <td style="width: 15%;" i18n="gpsSignalQty"></td>
-                                <td style="width: 65%;" i18n="gpsSignalStr"></td>
+                                <!--<td style="width: 10%;" i18n="gpsSignalGnssId"></td>-->
```

Comment:
Should this be removed?

Reply:
Actually, I forgot to ask about this. How does the localization system work? do I have to add gpsSignalGnssId with the "Gnss Id" string to the messages.json in all languages? In this case it's easy, but what about other strings that might need to be translated?

Reply:
Not, only English. We have translators that will do the rest of languages